src/components/command-bar/commands/search/components/custom-hits-container/index.tsx

```diff
@@ -21,8 +23,22 @@ const CustomHitsContainer = ({
 	noResultsSlot,
 	type,
 }: CustomHitsContainerProps) => {
+	const [hitCounts, setHitCounts] = useHitsContext()
```

Comment:
Is there some max hits in algolia for docs or tutorials? I noticed the numbers 20 (docs) & 25 (tutorials) coming up a lot when interacting with it. If there's a max hits setting, they should probably be consistent? Perhaps it was coincidence, but curious if you came across this.

Reply:
Ah great call-out, I didn't notice this! 🤔 I've added a task to our search board to investigate: 🎟 [Look into "max hits", might be different between Docs and Tutorials](https://app.asana.com/0/1204333060736722/1204613586844721/f)

I think it could make sense to handle the potential config updates separately from this PR, but open to taking a different approach 🙇‍♂️

Reply:
Okay, poked into this, it is indeed set to `20` for docs and `25` for tutorials. Have updated so they're both `25`. Will check in with product on this as well though, so leaving 🎟 [Look into "max hits", might be different between Docs and Tutorials](https://app.asana.com/0/1204333060736722/1204613586844721/f) open for now 👍